src/cli/opts.rs

```diff
@@ -221,20 +221,20 @@ pub enum Command {
         #[clap(long = "xmr-addr")]
         monero_address: XmrAddress,
 
-        /// An encoded public offer.
-        #[clap(short = 'o', long = "offer")]
-        public_offer: PublicOffer,
+        /// An encoded public deal.
+        #[clap(short = 'o', long = "deal")]
```

Comment:
I proposed in the wiki to use short `-D`

Reply:
916c4d5599a56c85111433daafe141f78906f882 :P